macros/SauceLabsPromo.ejs

```diff
@@ -0,0 +1,38 @@
+<%
+var version = (arguments.length) ? $0 : 'a';
+
+if (version === 'a') { %>
+  <% // runs on /docs/Tools %>
+  <div class="wiki-promo-sauce-labs" id="wiki-promo-sauce-labs-a">
+    <div class="wiki-promo-copy">
+      <p><strong>Learn about manual, automated and parallel testing with our Cross-browser Testing Tutorial.</strong></p>
```

Comment:
Guessing these strings are not being localised as they are here? https://github.com/mozilla/kuma/pull/4139/files#diff-141fd90a29db7f8735f7d16f5c45dd76R128